package.json

```
@@ -19,15 +19,15 @@
     "ipad-sm": "concurrently \"npx react-native run-ios --simulator=\\\"iPad Pro (11-inch) (4th generation)\\\" --mode=\\\"DebugDevelopment\\\" --scheme=\\\"New Expensify Dev\\\"\"",
     "start": "npx react-native start",
     "web": "scripts/set-pusher-suffix.sh && concurrently npm:web-proxy npm:web-server",
-    "web-proxy": "node web/proxy.js",
+    "web-proxy": "npx ts-node web/proxy.js",
```

```suggestion
    "web-proxy": "ts-node web/proxy.js",
```

Does putting `npx` make difference? I was able to run with only `ts-node`, could you test all scripts with only `ts-node`?

---

If you put npx it will prompt to install ts-node if you don't have it and it will run the desired script.
If you do not use npx and you do not have ts-node it just complains about command not being found.

---

I agree and changed it. Please have a look, thanks!